private/coursier_utilities.bzl

```diff
@@ -46,6 +46,10 @@ def strip_packaging_and_classifier(coord):
     return ":".join(coordinates)
 
 def strip_packaging_and_classifier_and_version(coord):
+    coordinates = coord.split(":")
```

Comment:
Do you mind adding a test to `tests/unit/coursier_utilities_test.bzl` for this?

Reply:
added a test case now